.github/workflows/test-and-lint.yml

```
@@ -0,0 +1,74 @@
+name: Test and Lint
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Set up node version
+        uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+          cache: 'yarn'
+
+      - name: Install Dependencies
+        run: yarn install --immutable --immutable-cache
+
+      - name: Run lint
+        run: ./node_modules/.bin/standard
+
+  test:
+    runs-on: ubuntu-latest
+    env:
+      DB_ENV_POSTGRESQL_DB: scale_of_belief
+      DB_ENV_POSTGRESQL_USER: postgres
+      DB_ENV_POSTGRESQL_PASS: postgres
+
+    services:
+      postgres:
+        # Docker Hub image
+        image: postgres:10
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
```

Any chance something like this works so you don't have to define them twice? They could need to be wrapped in curlies... but hopefully not.
```suggestion
          POSTGRES_USER: env.DB_ENV_POSTGRESQL_USER
          POSTGRES_PASSWORD: env.DB_ENV_POSTGRESQL_PASS
```